src/scripts.js

```diff
@@ -16,6 +16,7 @@ console.log('This is the JavaScript entry file - your code begins here.');
 import userData from './data/users';
 import User from './User';
 import UserRepository from './UserRepository';
+import Hydration from './Hydration'
 
 //=====================================================
 //imports?
```

Comment:
We have our imports above the space we labeled for imports, not a huge issue, but something to look at when we refactor